common/utils/src/main/resources/error/error-classes.json

```diff
@@ -4627,6 +4627,26 @@
     ],
     "sqlState" : "42KDF"
   },
+  "AGGREGATE_FILTER_EXPRESSION_NON_DETERMINISTIC" : {
```

Comment:
Please, create an error class w/ a common error message, and its sub-classes w/ specific error messages.

Reply:
just a thought but we could also have just one error with parametrized message

Reply:
If it is possible, let's do that. But we should avoid embedding text to source code or passing parts of error messages as parameters. Tech writers should correct error messages only in `error-classes.json`, and don't modify source code.

Reply:
I haven't found a way/example to do what Stefan suggested without having to change source code, that is, to do everything within `error-classes.json`. So, I went with what Max suggested.